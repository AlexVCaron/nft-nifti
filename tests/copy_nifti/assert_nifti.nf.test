nextflow_process {

  name "Test COPY_NIFTI"

  script "tests/copy_nifti/copy_nifti.nf"
  process "COPY_NIFTI"

  test("Should work with nifti") {

    when {
      process {
        """
        input[0] = file("$baseDir/tests/t1.nii.gz", checkIfExists: true)

        """
      }
    }
    then {
      assert compare(process.out.nifti.get(0))
    }

  }
}