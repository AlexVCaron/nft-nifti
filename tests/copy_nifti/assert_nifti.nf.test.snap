{
    "Test - nft-nifti - precision 2": {
        "content": [
            "output.nii.gz:md5,eb1bc18d187c12991d18a9b26175b071"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-16T09:48:18.39733413"
    },
    "Test - nft-nifti - precision 8": {
        "content": [
            "output.nii.gz:md5,df1eedb67d9511f0919a9f88d0031281"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-16T09:48:26.00410779"
    },
    "Test - nft-nifti - precision default": {
        "content": [
            "output.nii.gz:md5,df1eedb67d9511f0919a9f88d0031281"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-16T09:48:22.169359996"
    }
}